app/lib/rocketchat/services/restApi.ts

```diff
@@ -651,12 +651,12 @@ export const toggleFollowMessage = (mid: string, follow: boolean) => {
 };
 
 export const getThreadsList = ({ rid, count, offset, text }: { rid: string; count: number; offset: number; text?: string }) => {
-	const params: any = {
+	const params = {
 		rid,
 		count,
 		offset,
 		sort: { ts: -1 }
```

Comment:
The param `sort` is still used?

Reply:
I had not change this part, was already implemented... but I take a look at Rocket.Chat, and saw this `sort: sort || { tlm: -1 },`... so I removed the `sort` param and get the same result.

Reply:
Nice!